src/main/java/com/redhat/developers/msa/ola/OlaController.java

```diff
@@ -104,22 +104,35 @@ public String health() {
      * @return The feign pointing to the service URL and with Hystrix fallback.
      */
     private HolaService getNextService() {
-        // This stores the Original/Parent ServerSpan from ZiPkin.
-        final ServerSpan serverSpan = brave.serverSpanThreadBinder().getCurrentServerSpan();
-        final CloseableHttpClient httpclient =
-            HttpClients.custom()
-                .addInterceptorFirst(new BraveHttpRequestInterceptor(brave.clientRequestInterceptor(), new DefaultSpanNameProvider()))
-                .addInterceptorFirst(new BraveHttpResponseInterceptor(brave.clientResponseInterceptor()))
-                .build();
+        String url = System.getenv("HOLA_SERVER_URL");
```

Comment:
Yes. On OpenShift, we have the host and the port for a service "name". So, we would have the option to either have the *_SERVER_URL explicitly defined on Dockerfiles, or not use that and use the SERVICENAME_PORT and SERVICENAME_HOST that OpenShift makes available.